### PR TITLE
Proofread operators

### PR DESCRIPTION
**Pull request by yidawang**
Also fix a few typos and add two labels in previous sections.

### CODE REVIEW

File: chapter_common_operators/conv.md [hunk 1]
```diff
@@ -102,29 +110,39 @@ def conv(oc, ic, nh, nw, kh, kw, ph=0, pw=0, sh=1, sw=1):
...
+    constructor : user-defined tensor constructor
+    """
     np.random.seed(0)
     data = np.random.normal(size=(ic, n, n)).astype('float32')
     weight = np.random.normal(size=(oc, ic, k, k)).astype('float32')
     on = conv_out_size(n, k, p, s)
     out = np.empty((oc, on, on), dtype='float32')
-    if generator:
-        data, weight, out = (generator(x) for x in [data, weight, out])
+    if constructor is not None:
```
Comment (mli) [inline chapter_common_operators/conv.md]: any particular reason to change it? `if generator` is shorter

File: chapter_common_operators/matmul.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # Matrix Multiplication
 
-Matrix Multiplication is one of the most widely operators in scientific computing. Let's implement its computation in this chapter.
+Matrix Multiplication is one of the most widely operators in scientific computing and deep learning,which is typically referred to as *GEMM* (GEneral Matrix Multiply). Let's implement its computation in this section.
```
Comment (mli) [inline chapter_common_operators/matmul.md:3]: I'm fine to change chapter to section. If so, we need to update all md files.

File: chapter_common_operators/matmul.md [hunk 3]
```diff
@@ -26,11 +26,11 @@ def matmul(n, m, l):
     C : n x m matrix with C = A B
     """
     k = tvm.reduce_axis((0, l), name='k')
-    A = tvm.placeholder((n, l), name='A')
-    B = tvm.placeholder((l, m), name='B')
+    A = tvm.placeholder((n, l), name='a')
```
Comment (mli) [inline chapter_common_operators/matmul.md]: Upper cases are for symbolic tensors, lower cases are used by ndarrays.

File: chapter_common_operators/matmul.md (hunk 2 above)
Comment (yidawang) [reply chapter_common_operators/matmul.md:3]: Yes, I am updating it accordingly along the way. I think using section is better as it is consistent with the name generated by `:numref`. Chapter can be used when we refer to the entire unit, i.e. every fold in the repo.

File: chapter_common_operators/matmul.md (hunk 3 above)
Comment (yidawang) [reply chapter_common_operators/matmul.md]: Makes sense, will change them back.

File: chapter_common_operators/conv.md (hunk 1 above)
Comment (yidawang) [reply chapter_common_operators/conv.md]: It is for being consistent with the `get_abc` method in http://tvm.d2l.ai/chapter_getting_started/vector_add.html#implemeting-with-numpy

BTW, just caught another typo in the title above, will fix it soon as well :)

Comment (mli) [reply chapter_common_operators/conv.md]: we can remove `is not None`

Comment (yidawang) [reply chapter_common_operators/conv.md]: It is also for the consistency issue. Should we remove `is not None` in http://tvm.d2l.ai/chapter_getting_started/vector_add.html#implemeting-with-numpy as well? I am fine with either way.

**Pull request merged**